Python/chapter01/1.4 - PalinPerm/Nick4.py

```diff
@@ -15,15 +15,15 @@ def palperm(string):
         string (str): a string to check for all palindrome permutations
     '''
     table = {}
```

Comment:
this can be expressed more concisely with [`collections.defaultdict`](https://docs.python.org/3/library/collections.html#collections.defaultdict), and give it a more self-explanatory name while we're at it:

```suggestion
     letter_frequencies = collections.defaultdict(int)
     for letter in string.lower().replace(' ', ''):
       letter_frequencies[letter] += 1
```